### DISCUSSION TITLE
Soroban Banners

### DISCUSSION THREAD

**Discussion by theboycoder** [stellar/sorobanathon#149] [General]
[https://www.reddit.com/r/Stellar/comments/z410na/become_an_early_soroban_adopter_and_join/](https://www.reddit.com/r/Stellar/comments/z410na/become_an_early_soroban_adopter_and_join/)

[https://www.reddit.com/r/Stellar/comments/zebvup/smartcontract_platform_soroban_launches_a_100m/](https://www.reddit.com/r/Stellar/comments/zebvup/smartcontract_platform_soroban_launches_a_100m/)

![Sorobanathon_ First Light Banner](https://user-images.githubusercontent.com/45983304/207729856-1f8a54a9-cbe3-4db9-980c-a1ac01023aa3.png)

![Soroban Fund](https://user-images.githubusercontent.com/45983304/207729900-a2aae87f-d904-4384-864e-e5ebfd4c9956.png)

**Comment by stellarjohn** [comment]
Boxy!
